lib/daru/maths/statistics/vector.rb

```diff
@@ -7,6 +7,8 @@ module Statistics
       module Vector
         def mean
           @data.mean
+          rescue ZeroDivisionError
```

Comment:
I wanted a method to calculate sum without have to check if the element is nil every time so that the method is not slowed down as a result of the conditional.